releasenotes/notes/feat-electronic-denisty-initializations-d1bd5131c5776418.yaml

```diff
@@ -0,0 +1,19 @@
+---
+features:
+  - |
+    Three news means of initializing an :class:`~qiskit_nature.second_q.properties.ElectronicDensity`
+    have been added.
+
+    .. code-block:: python
+
+      size = 4  # number of spatial orbitals
+
+      from qiskit_nature.second_q.properties import ElectronicDensity
+
+      empty = ElectronicDensity.empty(size, include_rdm2=True)
```

Comment:
Since the API has num_spatial_orbitals as the name, not size, would it not be a little more straightforward to do this
```
num_spatial_orbitals = 4

empty = ElectronicDensity.empty(num_spatial_orbitals, include_rdm2=True)
```
or even
```
empty = ElectronicDensity.empty(num_spatial_orbitals=4, include_rdm2=True)
```
It ties things more closely to the names and avoids dealing with the indirect (size), which needs a comment here to clarify what it is, where the actual parameter name in the first place captures that intrinsically.